CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 ## [Unreleased]
 
+## [1.1.0] - 2022-09-26
```

Comment:
Could you add the following at the bottom　of CHANGELOG.
(Adding it, the version name will become the link of change between the previous version.)

```diff
 [Unreleased]: https://github.com/nhosoya/omniauth-apple/compare/v1.0.2...master
+[1.1.0]: https://github.com/nhosoya/omniauth-apple/compare/v1.0.0...v1.1.0
 [1.0.0]: https://github.com/nhosoya/omniauth-apple/compare/v0.0.3...v1.0.0
 [1.0.1]: https://github.com/nhosoya/omniauth-apple/compare/v1.0.0...v1.0.1
 [1.0.2]: https://github.com/nhosoya/omniauth-apple/compare/v1.0.1...v1.0.2
```

Reply:
probably you want this? https://github.com/nhosoya/omniauth-apple/pull/90/commits/492d85c3b06379dd39d32f07b70241b6c4cb72e9

Reply:
Oh, my comment was too appropriate, but thanks for taking my intent into account.